conda_smithy/configure_feedstock.py

```diff
@@ -1735,7 +1749,6 @@ def main(
 
         # Check that conda-smithy is up-to-date
         check_version_uptodate(r, "conda-smithy", __version__, True)
-        get_cfp_file_path(r, True)
```

Comment:
So removing this means that the no-check-update option no longer applies to the pinnings file?

Reply:
Yes, because cfp can never be out-of-date unless the cfp file is given as a path which didn't give a warning in master too.